Apps/WebClient/test/functional/e2e/cypress/integration/timeline/starRating.js

```diff
@@ -0,0 +1,20 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('Validate Star Rating', () => {
+    beforeEach(() => {
+    })
+
+    it('Cliking the 5 star button should logout', () => {
+        cy.login(Cypress.env('keycloak.username'), Cypress.env('keycloak.password'), AuthMethod.KeyCloak)
+        cy.get('[data-testid=logoutBtn]').click()
+        cy.get('.b-rating-star-empty:last').click()
```

Comment:
We can't add a testID for this?

Reply:
these stars are autogenerated by bootstrap.

Reply:
per our call, added the container testid of the b-rating-star-empty